apps/browser-extension-wallet/src/views/browser-view/features/assets/components/AssetsPortfolio/AssetsPortfolio.tsx

```
@@ -35,6 +38,16 @@ export interface AssetsPortfolioProps {
   isLoadingFirstTime?: boolean;
 }
 
+const searchTokens = (data: IAssetDetails[], searchValue: string) => {
+  const fields = ['name', 'policyId', 'fingerprint'] as const;
+  const lowerSearchValue = searchValue.toLowerCase();
+
+  return data.filter((item) =>
+    fields.some((field) => field in item && item[field] && item[field].toLowerCase().includes(lowerSearchValue))
+  );
+};
+
+// eslint-disable-next-line complexity
```

Please reduce the complexity instead of suppressing the error 🙏 . You could extract `useMemo` functions to the top of the file.

Or you could extract whole hooks
```
const useIsPortfolioBalanceLoading = ({ isBalanceLoading, portfolioTotalBalance }) => useMemo(
    () => new BigNumber(portfolioTotalBalance).isNaN() || isBalanceLoading,
    [isBalanceLoading, portfolioTotalBalance]
  );
...
const isPortfolioBalanceLoading = useIsPortfolioBalanceLoading({ isBalanceLoading, portfolioTotalBalance });
```

---

Improve AssetPortfolio component on 4060eae